### ISSUE TITLE
Installation from PSGallery fails

### ISSUE THREAD

**Issue by Swonkie** [Azure/azure-powershell#11417]
## Description

Installation as described here fails.

https://docs.microsoft.com/en-us/powershell/azure/install-az-ps?view=azps-3.6.1

Error:

```
PackageManagement\Install-Package : Cannot convert value "2.0.0-preview" to type "System.Version". Error: "Input
string was not in a correct format."
At C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1\PSModule.psm1:1809 char:21
+ ...          $null = PackageManagement\Install-Package @PSBoundParameters
+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (Microsoft.Power....InstallPackage:InstallPackage) [Install-Package], E
   xception
    + FullyQualifiedErrorId : InvalidCastParseTargetInvocation,New-PSGetItemInfo,Microsoft.PowerShell.PackageManagemen
   t.Cmdlets.InstallPackage
```

## Steps to reproduce

```powershell
Install-Module -Name Az -AllowClobber
```

## Environment data

```
Name                           Value
----                           -----
PSVersion                      5.1.17763.1007
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.1007
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by dingmeng-xue** [comment]
Please upgrade PowerShellGet to 2.2.x and try again.

**Comment by Swonkie** [comment]
PowerShellGet wasn't even installed on my machine when I tried this. After running `Install-Module -Name PowerShellGet -Force` I had version 1.0.0.1
To update to a newer version I first had to `Install-PackageProvider -Name NuGet -Force` and re-open the PowerShell session. Then I could finally update PowerShellGet to version 2.2.3.
Then I could successfully install the Azure module with `Install-Module -Name Az -AllowClobber`.

Thanks for the hint.
